modules/signatures/windows/virus_neshta.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, *args, **kwargs):
     def on_call(self, call, process):
         if call["api"] == "RegSetValueExA":
             key = self.get_argument(call, "FullName").lower()
-            if ".*\\software\\classes\\exefile\\shell\\open\\command.*" in key:
+            if r".*\\software\classes\\exefile\\shell\\open\\command.*" in key:

```

Comment:
This doesn't look like a regular expression, just is being used with `in` - so it should be `if r"\software\classes\exefile\shell\open\command" in key:` - right?